src/bin/flamegraph.rs

```diff
@@ -66,6 +70,9 @@ impl Into<Options> for Opt {
             options.title = "Icicle Graph".to_string();
         }
         options.negate_differentials = self.negate;
+        if let Some(factor) = self.factor {
+            options.factor = factor;
+        }
```

Comment:
Can't this just set `options.factor = self.factor`?

Reply:
No, because `options.factor` is `f64` and `self.factor` is `Option<f64>`. `option.factor` gets set to `1.0` in the `Default` implementation and we only override it here if `--factor` was passed on the command line.

Reply:
Hmm.. Why do we need it to be `Option<f64>` anywhere then? We already have a `default_value` set for it, right?

Reply:
We want the default to be `1.0` when using as a library, not just when using the CLI, so I have the default on the library's `Options` struct similar to the `title` field.  But I think you're right that we should use `default_value` on the `structopt` field. This just means that we have the default value defined in two places.